docs/source/upgrade-guide/index.md

```diff
@@ -30,6 +30,82 @@ the latest changes, and propose to you to merge the changes, so you can run it o
 
 ## Upgrading to Volto 16.x.x
 
+### `volto-slate` is now core
+
+From Volto 16.0.0-alpha.15 onwards, `volto-slate` is integrated into Volto core and enabled by default. This means that the default text block is now called `slate` and the former/deprecated text block `text` based on `draftJS` is now restricted (hidden) and enforced on Volto bootstrap. This is a major change and should be planned in advance before you install any version of the alphas after 16.0.0-alpha.15.
+
+````{note}
+Volto 16 is intended to be the LTS version used when Plone 6 Final is released. Volto 16 will ensure be compatible with all existing Volto deployments and won't introduce any hard breaking change. So, you'll have the choice to continue using `draftJS` text block in your projects. See below for more information.
+````
+
+```{important}
+From Volto 16 on, `draftJS` is deprecated and might be removed from core in next major versions.
+```
```

Comment:
Delete this `important` admonition, as I moved its content into the `deprecated` directive above.